test/local/tokens/session_token.js

```diff
@@ -19,7 +19,9 @@ const TOKEN = {
   email: Buffer('test@example.com').toString('hex'),
   emailCode: '123456',
   emailVerified: true,
-  tokenVerificationId: crypto.randomBytes(16)
+  tokenVerificationId: crypto.randomBytes(16),
+  verificationMethod: 2, // Totp verification method
```

Comment:
should this be `'totp-2fa'` ?

Reply:
or at least some string, it could be confusing

Reply:
Good point, I exposed a more readable version of `verficationMethod` in the another session value`verifcationMethodValue`, [ref](https://github.com/mozilla/fxa-auth-server/blob/15712f808084574759e6a314ed5088770ed4b345/lib/tokens/session_token.js#L12). I couldn't overwrite `verificationMethod` with the string value because there are tests that state a token can be recreated from the same token.